server/methods.js

```diff
@@ -675,4 +677,83 @@ Meteor.methods({
 
     return Accounts.createUser({ email, password, profile: { name: profileName } });
   },
+  createPDF(slug) {
```

Comment:
@singhsanket143 can you make sure here if the current user is admin?

Reply:
@peter-hank It is already handled

Reply:
> @peter-hank It is already handled

@singhsanket143 where exactly?

Reply:
@peter-hank The archive button is not rendered for a non-admin user. this is handled on the frontend

Reply:
@peter-hank I have also added logic to the backend now